### PR TITLE
Add device detection and limit amount of markup rendered.

### PR DESCRIPTION
**Pull request by alloy**
This pulls in Reaction v9 and updates the API usage to pass the user-agent so Reaction can detect what devices to render for.

### CODE REVIEW

**Comment by alloy** [comment]
@zephraph Ok, this should be good to 👍

File: package.json [hunk 1]
```diff
@@ -54,7 +54,7 @@
     "@artsy/express-reloadable": "1.4.0",
     "@artsy/palette": "2.23.0",
     "@artsy/passport": "1.1.0",
-    "@artsy/reaction": "8.2.0",
+    "@artsy/reaction": "^9.0.3",
```
Comment (zephraph) [inline package.json]: You can just pin this to 9.0.3. Renovate will handle bumping it.

File: src/desktop/apps/order2/routes.tsx [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { buildServerApp } from "reaction/Artsy/Router/server"
 import { buildServerAppContext } from "desktop/lib/buildServerAppContext"
 import { routes } from "reaction/Apps/Order/routes"
 import { stitch } from "@artsy/stitch"
-import metaphysics from "lib/metaphysics.coffee"
+const metaphysics = require("lib/metaphysics.coffee")
```
Comment (zephraph) [inline src/desktop/apps/order2/routes.tsx]: oh, this is interesting... what's happening here?

File: package.json (hunk 1 above)
Comment (zephraph) [reply package.json]: It's fine though. Renovate will PR the pin later.

File: src/desktop/apps/order2/routes.tsx (hunk 2 above)
Comment (damassi) [reply src/desktop/apps/order2/routes.tsx]: Typescript doesn't recognize the `.coffee` extension, but we've been conventionally getting around it with a normal require (not sure if there's a better way).

Comment (alloy) [reply src/desktop/apps/order2/routes.tsx]: Maybe we can define an ambient declaration for `*.coffee` and have it export `any` like the advice was for JSON files

Comment (zephraph) [reply src/desktop/apps/order2/routes.tsx]: I'd normally do a `ts-ignore`, but yeah, this works.

**Pull request merged**